tasks/onesky_import.js

```diff
@@ -18,88 +18,93 @@ module.exports = function (grunt) {
 
         var done = this.async();
 
+        var uploadFiles = [];
+
         var options = this.options({
             authFile: 'onesky.json',
             projectId: '',
             locale: '',
             file: '',
+            files: [],
             fileFormat: 'HIERARCHICAL_JSON',
             isKeepingAllStrings: true
         });
 
-        return upload();
-
-        ///////////////////////////
-
-        function upload() {
-            var api = getApi();
-            var url = api.baseUrl + api.path;
-
-            var form = new FormData();
-            form.append('api_key', api.publicKey);
-            form.append('timestamp', api.timestamp);
-            form.append('dev_hash', api.devHash);
-            form.append('file', fs.createReadStream(options.file));
-            form.append('file_format', options.fileFormat);
-            form.append('locale', options.locale);
-
-            if (_.isBoolean(options.isKeepingAllStrings)) {
-                form.append('is_keeping_all_strings', options.isKeepingAllStrings.toString());
-            } else {
-                grunt.fail.warn('Expected "options.isKeepingAllStrings" to be a boolean');
-            }
-
-            form.submit(url, onUpload);
-
-            function onUpload(error, response) {
-                if (error) { throw error; }
-
-                response.on('data', function (data) {
-                    data = JSON.parse(data);
-
-                    if (response.statusCode === 201) {
-                        onUploadSuccess(data);
-                    } else {
-                        onUploadError(data);
-                    }
-                });
-
-                response.resume();
-                done();
-            }
+        if (!!options.files.lenght) {
+            options.files.push({projectId: options.projectId, locale: options.locale, file: options.file});
```

Comment:
This line is only adding the `locale` and `file` defined at the top level of the options object. It does not match the example you specified in the README.md. 

Furthermore, it does not follow the standard of handling glob patterns (see grunt.file.expand..  http://gruntjs.com/api/grunt.file) to add multiple files. This means that the files options cannot be an array of objects. Instead it must be a single string with a glob pattern or an array of strings with file paths or glob patterns.

With this, we cannot support upload multiple files with different projectIds and locales in the same task.